### PR TITLE
Add auto-flake

### LINKED ISSUE #131: Fix darglint errors in tests
## Where?
We still have some darglint errors in **tests** files which needs to be fixed.
Also make make darglint check in  GitHub Actions **mandatory** (currently we ignore errors).

## How to check
Make sure you have darglint installed, do

`$ darglint tests`

You will see following errors
Screenshot 2021-04-28 at 11 20 30 PM

## Additional Context
Make sure you complete both of the tasks 😄

- [ ] Fix darglint errors.
- [ ] Make darglint mandatory in GitHub Actions.

### PR DESCRIPTION
**Pull request by jmaunon**
## Description
Add `autoflake` in pre-commit to remove unused imports and variables.

I have updated GA as well to close #131 
I have included the verbosity option to `darglint` to see errors message in the output of GA

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
I have tested including a `foo.py`

https://github.com/OpenMined/SyMPC/runs/2486952686?check_suite_focus=true

image

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by jmaunon** [comment]
@gmuraru

**Comment by Syzygianinfern0** [comment]
> I have tested including a foo.py

I'm sorry, but I don't seem to be able to see this file.

**Comment by gmuraru** [comment]
See the comment from PySyft - also, running with recursive, it might take some time to run the pre-commit scripts

**Comment by jmaunon** [comment]
> > I have tested including a foo.py
> 
> I'm sorry, but I don't seem to be able to see this file.

Ups, that's true. I do not see the commit, I will check later

**Comment by Syzygianinfern0** [comment]
> > I have tested including a foo.py
> 
> I'm sorry, but I don't seem to be able to see this file.

**Comment by jmaunon** [comment]
@Syzygianinfern0

**Review by Syzygianinfern0 — APPROVED** [review]
LGTM :raised_hands:

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -55,4 +55,15 @@ repos:
...
         exclude: ^tests/
+
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --recursive
+          - --remove-unused-variables
```
Comment (gmuraru) [inline .pre-commit-config.yaml:69]: NITPICK: Remove unused imports :smile:

**Review by gmuraru — APPROVED** [review]
Great! Thank you!

**Pull request merged**